tests/integration/test_z_scores.py

```diff
@@ -0,0 +1,88 @@
+from unittest import TestCase
+
+import numpy as np
+
+from .fixtures import FIXT_ADMIT_X_DEPT_UNWEIGHTED
+from .fixtures import FIXT_ADMIT_X_GENDER_WEIGHTED
+from .fixtures import FIXT_SELECTED_CROSSTAB_4
+from .fixtures import FIXT_MR_X_CAT
+from .fixtures import FIXT_CAT_X_MR
+
+from cr.cube.crunch_cube import CrunchCube
+
+
+class TestZScores(TestCase):
+    '''Test cr.cube implementation of Z-Scores.'''
+
+    def test_z_scores_admit_x_dept_unweighted(self):
+        '''Z-Scores for normal unweighted crosstab.'''
+        cube = CrunchCube(FIXT_ADMIT_X_DEPT_UNWEIGHTED)
+        expected = np.array([
+            [
+                18.04029231,
+                13.01839498,
+                -2.57673984,
+                -3.05952633,
+                -7.23024453,
+                -19.32141026,
+            ],
+            [
+                -18.04029231,
+                -13.01839498,
+                2.57673984,
+                3.05952633,
+                7.23024453,
+                19.32141026,
+            ],
+        ])
+        actual = cube.z_scores
+        np.testing.assert_almost_equal(actual, expected)
+
+    def test_z_scores_admit_x_gender_weighted(self):
+        '''Z-Scores for normal weighted crosstab.'''
+        cube = CrunchCube(FIXT_ADMIT_X_GENDER_WEIGHTED)
+        expected = np.array([
+            [9.42561985, -9.42561985],
+            [-9.42561985, 9.42561985],
+        ])
+        actual = cube.z_scores
+        np.testing.assert_almost_equal(actual, expected)
```

Comment:
remind me, is assert_almost_equal sufficiently precise by default? I vaguely recall that it was like 5 decimals and we should be able to count on at least 8 or 12, I would think

Reply:
The default is `decimal=7` in the function definition... I think we can increase the precision if we want to...